lib/account/pages/profile_page.dart

```diff
@@ -271,12 +277,12 @@ class _ProfilePageState extends State<ProfilePage> {
       if (!_profile!.isCurrentUser) {
         final bool hasRecentReport = _profile!.reportsReceived!
             .any((Report report) => report.isRecent && report.reporterId == SupabaseManager.getCurrentProfile()!.id);
-
```

Comment:
why do you hate proper spacing between lines? :D

Reply:
sorry ist bei debugging entstanden